### ISSUE TITLE
Add dd.mm.yyyy dateformat in superset-ui-chart-controls (Customize Tab)

### ISSUE THREAD

**Issue by csrinnuce** [apache/superset#19608]
## Screenshot

Availabe date formats in superset-ui-chart-controls now (master):

![grafik](https://user-images.githubusercontent.com/102681186/162387285-634840d8-6bd6-4e19-9361-fbccef0c58c9.png)

New date format proposal:

![dateformat](https://user-images.githubusercontent.com/102681186/162386896-2d85711f-0550-40b9-a9d6-dd247bb8e2fa.png)

## Description

26 countries use "dd.mm.yyyy" as date format.  (https://en.wikipedia.org/wiki/Date_format_by_country)

I'd like to add this format ("%d.%m.%Y "in D3 notation) below "Adaptive Formatting" in

https://github.com/apache/superset/blob/master/superset-frontend/packages/superset-ui-chart-controls/src/utils/D3Formatting.ts

**Comment by srinify** [comment]
Good idea! Moving to Discussions so we can keep Issues focused on Bugs and SIPs!
